qpext/cmd/qpext/main.go

```diff
@@ -37,6 +38,8 @@ import "knative.dev/serving/pkg/queue/sharedmain"
 
 var (
 	promRegistry *prometheus.Registry
+	EnvVars      = []string{"SERVING_SERVICE", "SERVING_CONFIGURATION", "SERVING_REVISION"}
+	LabelKeys    = []string{"service_name", "configuration_name", "revision_name"}
```

Comment:
slices instead of maps to keep order of label insertion - otherwise testing is a pain